src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -171,11 +171,16 @@
         </div>
         <!-- Applications -->
         <div class="form-group row">
-          <label i18n
-                 class="cd-col-form-label"
-                 for="applications">Applications</label>
+          <label class="cd-col-form-label required"
+                 for="applications">
+            <ng-container i18n>Applications</ng-container>
+            <cd-helper>
```

Comment:
Let's use <cd-help-text> after the input tag here as well. 
https://github.com/ceph/ceph/blob/ef9bf7651800632d10bacef3c41f3432db769f40/src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html#L252-L254 

We need to gradually move to this style, for all new things added and rest will be taken care by carbon

Thanks!

Reply:
I've already addressed this here https://github.com/ceph/ceph/pull/51591#. So once that's merged I can simple rebase this PR. Marking this PR as DNM for now and after https://github.com/ceph/ceph/pull/51591#

Reply:
@afreen23 I've merged  https://github.com/ceph/ceph/pull/51591 and rebased this PR, can you review it again?